packages/core/src/browser/keybinding.ts

```diff
@@ -540,14 +540,14 @@ export class KeybindingRegistry {
 
     /**
      * Match first binding in the current context.
-     * Keybinidngs ordered by a scope and by a registration order within the scope.
+     * Keybindings ordered by a scope and by a registration order within the scope.
      *
      * FIXME:
-     * This method should run very fast since it happens on each keystoke. We should reconsider how keybindings are stored.
-     * It should be possible to look up full and partial keybinding for given key sequnce for constant time using some kind of tree.
+     * This method should run very fast since it happens on each keystroke. We should reconsider how keybindings are stored.
+     * It should be possible to look up full and partial keybinding for given key sequence for constant time using some kind of tree.
      * Such tree should not contain disabled keybindings and be invalidated whenever the registry is changed.
      */
-    matchKeybiding(keySequence: KeySequence, event?: KeyboardEvent): KeybindingRegistry.Match {
+    matchKeybinding(keySequence: KeySequence, event?: KeyboardEvent): KeybindingRegistry.Match {
```

Comment:
Please mention this change in the changelog. Downstream consumers will have a behavior change if they've overridden this method.

Or keep the original method with the typo as deprecated for at least one release and delegate into the new one.